keyboards/dz60/keymaps/MintCollie/keymap.c

```diff
@@ -0,0 +1,51 @@
+#include QMK_KEYBOARD_H
+
+// Make special keycodes more visible
+#define ____ KC_TRNS
+#define XXXX KC_NO
+
+// Layer definition
+#define L0 0
+#define L1 1
+//#define L2 2
+
+enum custom_keycodes {
+	QMKBEST = SAFE_RANGE,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+	switch (keycode) {
+		case QMKBEST:
+			if (record->event.pressed) {
+				// when keycode QMKBEST is pressed
+				SEND_STRING("QMK is the best thing ever!");
+			} else {
+				// when keycode QMKBEST is released
+				register_code(KC_ENT);
+				unregister_code(KC_ENT);
+			}
+			break;
+	}
+	return true;
+};
```

Comment:
Unless it's actually needed/used, this can and should be removed.  it's included in the template, mostly as an example+placeholder. 

```suggestion
```